src/colmap/controllers/incremental_pipeline_test.cc

```diff
@@ -600,48 +655,53 @@ TEST(IncrementalPipeline, SfMWithRandomSeedStability) {
     return reconstruction_manager;
   };
 
-  // Single-thread execution
+  // Single-threaded execution.
```

Comment:
Needed to update the below logic to make the test results more reliable, because the newly added test was randomly breaking the below due to different random numbers in the generated synthetic scenes.